README.md

```diff
@@ -40,7 +40,7 @@ If unspecified, the app will use your local court-case-service app but you can a
  
 Start the standalone WireMock server
 ```
-java -jar wiremock-standalone-2.26.0.jar --global-response-templating --port 9091
+java -jar wiremock-standalone-2.26.2.jar --global-response-templating --port 9091
```

Comment:
Could you add instructions on how to start the Redis server etc? There's a few moving parts involved in running  the integration tests now so we ought to document it, would be nice to get it running from one command if possible though that's a bit more involved.

Reply:
I've updated the README with instructions on how to install Redis.
This will most likely be removed and setup with Docker when we look at this properly as part of PIC-304

Reply:
Edit. I've included docker-compose which starts a Redis server for simplicity.
I've updated the README accordingly.